tslint.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@eth-optimism/dev/tslint.json",
   "rules": {
     "array-type": false,
-    "class-name": false
+    "class-name": false,
+    "prefer-conditional-expression": false
```

Comment:
I think conditional expressions are pretty dumb

Reply:
Agree, not a fan of this syntax except for very simple cases

Reply:
Can this go into `@eth-optimism/dev` and then we update the dep in another PR?

Reply:
Change to `tslint.json` was included in this PR because tslint was complaining about a file and wanted to make the file *way* uglier.